Resources/ServerInfo/_NF/Guidebook/StartingGear.xml

```diff
@@ -0,0 +1,33 @@
+<Document>
+  # Starting Equipment
+  <Box>
+    <GuideEntityEmbed Entity="ClothingBackpackPilot"/>
```

Comment:
If this is for new players, why not use the grey backpack, since it'll probably actually be in their HUD?
```suggestion
    <GuideEntityEmbed Entity="NFClothingBackpack"/>
```

Reply:
> If this is for new players, why not use the grey backpack, since it'll probably actually be in their HUD?

Missed this, yeah only reason I used pilot is cause I was a pilot character and didnt know what the grey one was called.